src/lib.rs

```diff
@@ -31,12 +31,14 @@ pub mod net;
 
 #[cfg(all(unix, feature = "rust_tls"))]
 pub mod rust_tls;
+
 #[cfg(all(unix, feature = "rust_tls", not(feature = "native2_tls")))]
 pub use rust_tls as tls;
 
-#[cfg(all(unix, feature = "native2_tls"))]
+#[cfg(all(any(unix, windows), feature = "native2_tls"))]
 pub mod native_tls;
-#[cfg(all(unix, feature = "native2_tls", not(feature = "rust_tls")))]
+
+#[cfg(all(any(unix, windows), feature = "native2_tls", not(feature = "rust_tls")))]
```

Comment:
This was needed to get k8-client working over in https://github.com/infinyon/fluvio/pull/1218 but this breaks CI here :/